app/src/main/java/com/github/sdpcoachme/ui/Dashboard.kt

```diff
@@ -279,35 +326,87 @@ fun AppBar(title: @Composable (Modifier) -> Unit, onNavigationIconClick: () -> U
     )
 }
 
+/**
+ * Composable that displays the header of the drawer.
+ * It contains the user's profile picture, name and email.
```

Comment:
Great idea to add javadocs here, this makes it much more clear what to expect from this composable!